src/constants/schema.ts

```diff
@@ -146,6 +146,24 @@ export const tenantInvitationSchema = z.object({
 
 export type TTenantInvitationSchema = z.infer<typeof tenantInvitationSchema>;
 
+/**
+ * The schema for the shareHouse creation form
+ */
+export const shareHouseCreationSchema = shareHouseNameSchema.extend({
+  startDate: z.string(),
```

Comment:
Could you add explicit DateTime validation to ensure it follows ISO 8601 like below?

https://zod.dev/?id=datetimes

```suggestion
  startDate: z.string().datetime(),
```